src/Mod/PartDesign/Gui/TaskHoleParameters.ui

```diff
@@ -258,7 +258,7 @@
      </property>
     </widget>
    </item>
-   <item row="22" column="2">
+   <item row="23" column="2">
```

Comment:
Why does this row number need to change?

Reply:
Good eye.  Initially I had the Reversed checkbox below the Tapered checkbox, and I had moved the vertical spacer down to the next row, but then I decided to put the Reversed checkbox on the same row as the Tapered checkbox so users would be less likely to need to scroll down as it's already a fairly tall dialog.  When I moved the Reversed checkbox up I guess I forgot to also move the spacer back up to where it was originally.

Reply:
👍 I doubt the row number actually makes a difference, but it's probably worth changing it back to avoid confusing future devs.